test/config/MockAppConfig.scala

```diff
@@ -22,7 +22,7 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 class MockAppConfig extends MockFactory {
 
-  def config(encrypt: Boolean = true): AppConfig = new AppConfig(mock[ServicesConfig]) {
+  def config(encrypt: Boolean = true, enableTailoring: Boolean = false): AppConfig = new AppConfig(mock[ServicesConfig]) {
```

Comment:
this im unsure on i think its ok, but maybe we could just make a new config(tailoring config) below where we just copy this one and overwrite tailoring. Although if we just have the tailoring config set to true by default as mentioned above this would not even matter.